Dockerfile

```diff
@@ -9,14 +8,12 @@ RUN apt-get update && apt-get -y upgrade && \
         # git is required for installing gems from git repos
         git \
         # libjemalloc1 (v3) provides big memory savings vs jemalloc v5+ (default on debian buster)
-        libjemalloc1 \
         libpq-dev \
-        nodejs \
+        postgresql-client-11 \
         tmpreaper \
         && \
         apt-get clean
 
-ENV LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so.1
```

Comment:
this is for dynamically switching the ruby memory allocator at run time - if we switch back to jemalloc v1 we will need this back.